docs/REST API/rest-api.md

```diff
@@ -9152,6 +9152,12 @@ Returns information about the specified software. By default, `versions` are sor
       "post_install_script": "sudo /Applications/Falcon.app/Contents/Resources/falconctl license 0123456789ABCDEFGHIJKLMNOPQRSTUV-WX",
       "uninstall_script": "/Library/CS/falconctl uninstall",
       "self_service": true,
+      "automatic_install_policies": [
+        {
+          "policy_id": 343,
+          "policy_name": "[Install software] Crowdstrike Agent",
+        } 
+      ], 
```

Comment:
To clarify, this should only show up when this endpoint is called with a team_id query parameter, correct?

In the UI, we have an All Teams view for a software title and a team view. For example:
- Team view: https://dogfood.fleetdm.com/software/titles/2792?team_id=2
- All Teams view: https://dogfood.fleetdm.com/software/titles/2792

Reply:
We just discussed this during estimation meeting, we won't see these when All teams is selected, since both `software_package` and `app_store_app` objects will be `null`.